ouroboros-consensus/test-consensus/Test/Consensus/Node.hs

```diff
@@ -136,38 +147,147 @@ test_checkProtocolMagicId_empty = res @?= Left e
   lockDb
 -------------------------------------------------------------------------------}
 
-test_lockDb :: Assertion
```

Comment:
State that we use the mock lock for testing that we can release and reacquire the lock, and the real lock for testing that we can't acquire the same lock twice at the same time.